<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Face</class>
 <widget class="QWidget" name="Face">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>680</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>680</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>重置图像</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>550</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>坐标:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_first">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>680</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>请用鼠标单击选择需要调整的中间位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>620</y>
     <width>231</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>调整范围</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_radius"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
